wp-content/themes/reconasia/inc/template-tags.php

```diff
@@ -188,6 +188,17 @@ function reconasia_last_updated() {
  *
  */
 function reconasia_authors() {
+
+	/*
+	* This will show or not the author information. If ACF hide_author_info is checked won't show the author.
+	*/
+	$hide_author_info = get_field('hide_author_info');
+	if ( class_exists('ACF') ) {
```

Comment:
This should be moved up one line - `get_field` is an ACF function, so if ACF isn't installed, this will error out on line 195.

Reply:
Thank you for the clarification!